src/vis/image.js

```diff
@@ -239,7 +239,7 @@
       this.options.tiler_protocol = protocol;
       this.options.tiler_port     = port;
 
-      if (this.userOptions.https || this.imageOptions.vizjson.indexOf("https") === 0) {
```

Comment:
this is not needed since if the viz.json is requested by https the url it uses for the tiler is https (since 3.14 version)